src/components/__tests__/components/common/Input.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Input from '../../../common/Input/Input';
+
+describe('<Input />', () => {
+  test('Should render the <Input />', () => {
+    const handleInput = () => true;
+    const wrapper = shallow(<Input onChange={handleInput} />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Are you all using snapshots to test?

Reply:
@kimotho-njoki Yes, we are all using snapshots to test component rendering